scripts/json/node.json

```diff
@@ -0,0 +1,21 @@
+{
+  "host_info": {
+    "local_ip": "10.213.43.188",
+    "mac_address": "90:17:ac:c1:34:64",
+    "node_id": "9192a4d4-ffff-4ece-b3f0-8d36e3d85001",
+    "node_name": "node2",
+    "server_port": 8080,
+    "veth": "eth0"
+  }
+}
+
+{
+  "host_info": {
+    "local_ip": "10.213.43.166",

```

Comment:
For container deployment, 
"local_ip":"127.0.0.1"
"node_name":"local_node"
"server_port":50001

I think ACA opens port 50001 for gRPC connection, right? Does DPM consume the port for now?

Reply:
yes, dpm consume the port

Reply:
Based on today's discussion, DPM doesn't consume this port for now and the next step for us is (1) update DPM contract to include this field (2) DPM implementation and test update. @haboy52581

Reply:
so do you mean we have to change the contract now or after the release

Reply:
You can create a tracking issue and address the issue in a follow-up PR.